src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -553,8 +553,7 @@ private Symbol ResolveMultipleSymbolsInAttributeTypeLookup(ArrayBuilder<Symbol>
                 var mdSymbol = symbols[secondBest.Index];
 
                 //if names match, arities match, and containing symbols match (recursively), ...
-                if (srcSymbol.ToDisplayString(SymbolDisplayFormat.QualifiedNameArityFormat) ==
```

Comment:
>SymbolDisplayFormat.QualifiedNameArityFormat [](start = 46, length = 44)

Is it possible that SymbolDisplay tries to ask more information than it really needs for this format? Would it make sense to open an issue to follow up on that?